homework/zadanie4_five_philosophers.cpp

```diff
@@ -0,0 +1,104 @@
+#include <iostream>
+#include <thread>
+#include <vector>
+#include <chrono>
+#include <mutex>
+#include <string>
+#include <mutex>
+
+using namespace std::chrono_literals;
+
+std::mutex mtx;
+
+class CTable{
+
+public:
+    std::vector<bool> forks_pool;
```

Comment:
czemu nie vector mutexów? W wymaganiach tak było.

Reply:
Bo to bylo prapra-rozwiazanie:) teraz mamy clase CFork, ktora posiada mutex. Wreszcie mamy vector.